### ISSUE TITLE
Aliexpress product is different than what is shown on waveshare's website

### ISSUE THREAD

**Issue by brentpicasso** [upiir/custom_boost_gauge_esp32#2]
The aliexpress link shows a different base board with one connector instead of the two black connectors, as shown on your site and the waveshare wiki. https://www.aliexpress.us/item/3256805983216570.html

![image](https://github.com/upiir/custom_boost_gauge_esp32/assets/3267533/e21d5849-55c8-40dc-ac14-b66e78191131)

Which one is the current revision? or are these actually two different products? 

Also, is this display available by itself, without the controller board? What is the controller used for this display?

Thank you very much, your video was great by the way!

**Comment by upiir** [comment]
Yes, those are two separate products - with and without a touchscreen. Otherwise, they should be the same. 

The screen alone without microcontroller is here - https://www.waveshare.com/1.28inch-lcd-module.htm

Let me know if was helpful.
